src/Competition/Participant.java

```diff
@@ -0,0 +1,8 @@
+package Competition;
+
+public interface Participant {
```

Comment:
it needs to be class, Participant has attributes like max height and max length. The interface doesn't allow to have attributes. And getter is looking weird in the interface